### ISSUE TITLE
Create exam time formatting

### ISSUE THREAD

**Issue by jeremiah-ang** [HopeNUS/examtt#6]
Currently if the exam is 9am it'll send `DD/MM/YYYY 9:00` whereas there should be a leading 0 infront of the 9. (i.e. `DD/MM/YYYY 09:00`)

**Comment by jeremiah-ang** [comment]
In [/scripts/examTtExtractor.js](https://github.com/HopeNUS/examtt/blob/7b157809cd1576a464c61b352a98831cd4c22f7e/scripts/examTtExtractor.js), can make the following changes

```diff
+ const hour_formatted = extractExamTtHour(m['hour']);
const module = {
            code: m['code'],
            date: extractExamTtDate(m['date']),
            month: extractExamTtMonth(m['month']),
-           hour: extractExamTtHour(m['hour']),
+           hour: hour_formatted,
            minute: m['minute'],
            location: m['venue'],
            location_index: LOCATIONS[m['venue']],
-           datetime: `${m['date']}/${m['month']}/${YEAR} ${m['hour']}:${m['minute']}`,
+           datetime: `${m['date']}/${m['month']}/${YEAR} ${hour_formatted}:${m['minute']}`,
        }
```

**Comment by jeremiah-ang** [comment]
Fixed by b96001d55c44ddf13510398ddedec1a070032ae2
